PlaceRouteHierFlow/PnRDB/datatype.h

```diff
@@ -286,6 +286,7 @@ struct blockComplex {
 };
 
 struct PowerGrid{
+  std::string name; 
   vector<Metal> metals;
   vector<Via> vias;
   bool power = 1;//1 is vdd, 0 is gnd;
```

Comment:
Do we use this power field anymore? It seems to always be set to 1.

Reply:
The power field is not used.